test/unit/bdd_test.js

```diff
@@ -45,18 +45,18 @@ describe('BDD', () => {
     // console.log('Feature', ast.feature);
     // console.log('Scenario', ast.feature.children);
     // console.log('Steps', ast.feature.children[0].steps[0]);
-    assert.ok(ast.feature);
-    assert.ok(ast.feature.children);
-    assert.ok(ast.feature.children[0].steps);
+    expect(ast.feature).is.ok;
+    expect(ast.feature.children).is.ok;
+    expect(ast.feature.children[0].steps).is.ok;
   });
 
   it('should load step definitions', () => {
     Given('I am a bird', () => 1);
     When('I fly over ocean', () => 2);
     Then(/I see (.*?)/, () => 3);
-    assert.equal(1, matchStep('I am a bird')());
-    assert.equal(3, matchStep('I see ocean')());
-    assert.equal(3, matchStep('I see world')());
+    expect(1).is.equal(matchStep('I am a bird')());
+    expect(3).is.equal(matchStep('I see ocean')());
+    expect(3).is.equal(matchStep('I see world')());
```

Comment:
expect(CHECKED VALUE).TO.equal(EXPECTED VALUE)

`expect(matchStep('I see world')()).to.equal(3);`
or `expect(matchStep('I see world')()).is.equal(3);`